### ISSUE TITLE
Question about the experiment

### ISSUE THREAD

**Issue by infiniteHY** [JiakangYuan/HelixFormer#4]
Hello, I try to reproduce the result on CUB dataset (conv4 and 5-way 5-shot)
I got a bad result. Is my configuration setting wrong? or I miss something? 
Look forward to your help, Thank you!

> dataset: torch.Size([3, 84, 84]) (x2965), 50
> 
> 5
> num params: 112.8K
> test epoch 1: acc=67.04 +- 0.50 (%), loss=1.7668 
> test epoch 2: acc=67.13 +- 0.35 (%), loss=1.7614 
> test epoch 3: acc=67.23 +- 0.29 (%), loss=1.7555 
> test epoch 4: acc=67.36 +- 0.25 (%), loss=1.7496 
> test epoch 5: acc=67.41 +- 0.22 (%), loss=1.7460 

The following is my configuration:

```
train_dataset: CUB_crop
train_dataset_args: {split: train, augment: new_crop}
fs_dataset: CUB_crop
fs_dataset_args: {split: test}
eval_fs_epoch: 5

model: classifier
model_args:
    encoder: convnet4_21
    encoder_args: {}
    classifier: linear-classifier
    classifier_args: {n_classes: 100}
    mode: flatten

batch_size: 128
max_epoch: 100
optimizer: sgd
optimizer_args: {lr: 0.01, milestones: [85]}

save_epoch: 5
visualize_datasets: true
```

```
train_dataset: CUB_crop
train_dataset_args: {split: train, augment: new_crop}
tval_dataset: CUB_crop
tval_dataset_args: {split: test}
val_dataset: CUB_crop
val_dataset_args: {split: val}

encoder: convnet4_21
encoder_args: {}

relation_net: relation_net
relation_net_args: {input_size: [64, 21, 21]}

# trans_net: conv_helix_transformer/linear_helix_transformer
trans_net: conv_helix_transformer
trans_net_args:   {dim: 64, emb_dim: 64, emb_kernel: 3, emb_stride: 1,
                   proj_kernel: 3, kv_proj_stride: 1, heads: 2, dim_head: 64,
                   depth: 1, mlp_mult: 4, dropout: 0.}

# pretrained backbone
load_encoder: ./save/classifier_CUB_crop_convnet4_21/epoch-100.pth

# 5shot setting
n_way: 5
n_shot: 5
n_query: 15
n_train_way: 5
n_train_shot: 5
n_train_query: 15
train_batches: 250
ep_per_batch: 2
max_epoch: 150

optimizer: adam
optimizer_args: {lr: 0.001,  milestones: [80, 120]}

optimizer_rn: adam
optimizer_rn_args: {lr: 0.001, milestones: [80, 120]}

optimizer_trans: adam
optimizer_trans_args: {lr: 0.001, milestones: [80, 120]}

visualize_datasets: True
```

```
dataset: CUB_crop
dataset_args: {split: test}

encoder: convnet4_21
encoder_args: {}

relation_net: relation_net
relation_net_args: {input_size: [64, 21, 21]}

trans_net: conv_helix_transformer
trans_net_args: {dim: 64, s1_emb_dim: 64, s1_emb_kernel: 3, s1_emb_stride: 1,
                   s1_proj_kernel: 3, s1_kv_proj_stride: 1, s1_heads: 2, s1_dim_head: 64,
                   s1_depth: 1, s1_mlp_mult: 4, dropout: 0.}

load: ./save/Meta_train_helix-transformer_conv_helix_transformer_2heads_CUB_crop-5shot_1.12_convnet4_21/max-va.pth
load_relation: ./save/Meta_train_helix-transformer_conv_helix_transformer_2heads_CUB_crop-5shot_1.12_convnet4_21/max-va.pth
load_trans: ./save/Meta_train_helix-transformer_conv_helix_transformer_2heads_CUB_crop-5shot_1.12_convnet4_21/max-va.pth

```
